GlobalBehaviorandComponents/templates/components/template-login-widget-oie.html

```diff
@@ -120,53 +120,11 @@
                       },
                       i18n: {
                         en: {
-                          'primaryauth.title': "Sign in to "+{{dumps(config.settings.app_name)|safe}}
+                          'primaryauth.title': "Sign in to {{config.settings.app_name|safe}}"
```

Comment:
@srecinto So how are resolving it now? If not you we are bringing back #456.

Reply:
The dumps broke a bunch of things in other screens and was not the right approach for the way config stuff gets pushed out to the screen. I added a simple handler in the config mappings because it would globally fix that issue and not require dumps to do so

https://github.com/udplabs/zartan/pull/477/files#diff-4e750ec69d99e9c33434d3d56ec61ea7ac611e4a5a29bd65a6b157434a7eadc3R249